src/content/migrate/5.md

```diff
@@ -16,68 +16,55 @@ This guide aims to help you migrating to webpack 5 when using webpack directly.
 
 ## Preparations
 
-webpack 5 requires at least Node.js 10.13.0 (LTS).
+Webpack 5 requires at least Node.js 10.13.0 (LTS), so make sure you upgrade your Node.js if you're still running an older one.
 
-T> Using newer Node.js version can improve build performance.
+## Upgrade webpack 4 and its plugins/loaders
 
-### Upgrade webpack and its dependencies
+1. Upgrade `webpack` 4 to the latest available version.
 
-#### Upgrade webpack 4 to the latest available version
-
-When using webpack >= 4, upgrading to the latest webpack 4 version should not require additional guidance.
-If you are using webpack version less than 4 please see the [webpack 4 migration guide](/migrate/4/).
-
-#### Upgrade webpack-cli to the latest available version (when used)
-
-#### Upgrade all used Plugins and Loaders to the latest available version
+    - When using webpack >= 4, upgrading to the latest webpack 4 version should not require additional guidance.
```

Comment:
Might be useful to add a command for it? `npm upgrade webpack@4.44.2`?

Reply:
Not necessary, users should already know how to do it. Otherwise we would teach them to upgrade `webpack-cli` too.

Reply:
I don't agree with it. If users are already using webpack 4 and trying to upgrade webpack 5, they should obviously know how to upgrade to the latest webpack 4. I don't think people don't even know how to upgrade to the latest webpack 4 would read this migration.

Reply:
No, we don't know what is the knowledge in teams.

In any case, there is already a link to the v4 migration, so both versions are covered.